shell/common/shorebird.cc

```diff
@@ -52,11 +52,6 @@ void SetBaseSnapshot(Settings& settings) {
   // the mappings alive.
   vm_snapshot = DartSnapshot::VMSnapshotFromSettings(settings);
   isolate_snapshot = DartSnapshot::IsolateSnapshotFromSettings(settings);
-  assert(vm_snapshot);
```

Comment:
Moved this out of `SetBaseSnapshot` and into `ConfigureShorebird` since `SetBaseSnapshot` appears to never be called in Debug.